pixi.toml

```diff
@@ -0,0 +1,378 @@
+eyes-images = "==5.23.*"
+
+# Core packages required at runtime
+[feature.runtime.dependencies]
+# Core Python and utilities
+python = "3.12.*"
+pip = "*"
+numpy = "1.26.*"
+scipy = "1.14.*"
+numba = "0.61.*"
+numexpr = "2.8.*"
+requests = "2.32.*"
+psutil = "5.9.*"
+pyyaml = "*" 
+
+# Data I/O and file handling
+h5py = "3.13.*"
+hdf5 = ">=1.14.2,<1.15"
+tifffile = "2023.7.18.*"
+imagecodecs = "2023.9.18.*"
+
+# Image processing and scientific libraries
+astropy = "7.1.*"
+scikit-image = "0.25.*"
+jenkspy = "0.4.*"
+
+# Tomography and reconstruction
+algotom = "1.6.*"
+tomopy = "1.12.*"
+cil = "25.0.*"
+ccpi-regulariser = "*"
+ccpi-viewer = "25.0.*"
+
+# Visualization and GUI
+vtk = { version = "*", build = "qt*" }
+pyqt = "5.15.*"
+qt-main = "5.15.8.*"
+pyqtgraph = "0.13.7.*"
+qt-material = "2.14.*"
+darkdetect = "0.8.0.*"
+
+# GPU-specific packages - common across all platforms
+[feature.runtime-gpu.dependencies]
+cudatoolkit = "11.8.*"
+cupy = "12.3.*"
+
+# Linux-specific GPU packages - Linux (CUDA-enabled builds)
+[feature.runtime-gpu.target.linux-64.dependencies]
+astra-toolbox = { version = "2.1.0.*", build = "cuda*" }
+libastra = { version = "2.1.0.*", build = "cuda*" }
+
+# Windows-specific GPU packages - Windows (CUDA-enabled builds with specific hashes)
+[feature.runtime-gpu.target.win-64.dependencies]
+astra-toolbox = { version = "2.1.0.*", build = "cudapy312h0f47501_105" }
+libastra = { version = "2.1.0.*", build = "cuda11.8hb8fb34d_111" }
+
+# CPU-specific packages (non-CUDA builds)
+[feature.runtime-cpu.target.linux-64.dependencies]
+astra-toolbox = "2.1.0.*"
+libastra = "2.1.0.*"
+
+# Windows-specific CPU packages - Windows (non-CUDA builds with specific hashes)
+[feature.runtime-cpu.target.win-64.dependencies]
+astra-toolbox = { version = "2.1.0.*", build = "py312h6965d8f_5" }
+libastra = { version = "2.1.0.*" }
+
+# GPU verification feature - adds __cuda requirement - needed to ensure GPU builds only go to GPU-capable systems
+[feature.gpu-verify.system-requirements]
+cuda = "11.8"
+
+# Virtual package to ensure CUDA is available on GPU systems - needed for windows
+[feature.gpu-verify.target.win-64.dependencies]
+__cuda = "11.8.*"
+
+# Packages required for building and uploading conda packages
+[feature.build.dependencies]
+boa = "*"
+anaconda-client = "*"
+conda-build = "*"
+
+# Set environment variables for development environment during environment activation
+[feature.dev.activation.env]
+SOURCE_DIRS = "mantidimaging scripts docs/ext/"
+APPLITOOLS_API_KEY = "local"
+CONDA_DLL_SEARCH_MODIFICATION_ENABLE = "1"
+
+# Set environment variables for build environment during environment activation
+[feature.build.activation.env]
+UPLOAD_USER = "mantidimaging"
+AUTHENTICATION_PARAMS = "--user $UPLOAD_USER --token $ANACONDA_API_TOKEN"
+
+# Linux specific dependencies (to get Qt platform theme working)
+[target.linux-64.dependencies]
+qt-gtk-platformtheme = { version = "5.15.8.*", build = "he21a4df_1" }
```

Comment:
Do we need to specify a `build` here. (I know this was a pain point, so happy to accept yes)

Reply:
It seems to be the case yes, I can double check though. As I would like to minimise any instances where we need to do this as much as possible.

Reply:
from another round of testing, we can indeed remove build 🥳